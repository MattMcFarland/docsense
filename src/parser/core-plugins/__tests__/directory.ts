import dirPlugin from '../directory';
import loadPlugin from './utils/loadPlugin';

describe('core-plugin: directory', () => {
  test('matches snapshot', async () => {
    const runTest = loadPlugin({}, '__TEST__');
    const state = await runTest(dirPlugin, 'foo()');
    expect(state).toMatchSnapshot();
  });
});
