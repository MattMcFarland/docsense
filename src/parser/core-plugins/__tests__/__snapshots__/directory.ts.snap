// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core-plugin: directory matches snapshot 1`] = `
Object {
  "directory_collection": Array [
    Object {
      "directory_id": "",
      "files": Array [
        Object {
          "file_id": "L19fVEVTVF9f",
        },
      ],
    },
  ],
  "sourceCode": "foo()",
}
`;
